guard/src/commands/validate.rs

```diff
@@ -146,7 +233,8 @@ pub fn validate_and_return_json(
                 Ok(root) => {
                     let root_context = RootScope::new(&rules, &root);
                     let stacker = StackTracker::new(&root_context);
-                    let reporter = ConsoleReporter::new(stacker, "lambda-function", "input-payload", true, true, false);
+                    let reporters = vec![];
+                    let reporter = ConsoleReporter::new(stacker, &reporters, "lambda-function", "input-payload", true, true, false);
```

Comment:
Is `validate_and_return_json` function required in `validate.rs`? Because for guard-lambda, it is using `validate_and_return_json` function in `helper.rs`?

Reply:
@shreyasdamle yes this is part that we will re-visit post PR merge to expose this functionality via Lambda as well. This is in addition to the multiple rule functionality that you added.